module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,16 @@
+#include "sort.hpp"
+
+#include <algorithm>
+#include <forward_list>
+#include <string>
```

Comment:
You have included string in the header,so it's redundant here.

Reply:
+1